src/main/cpp/t08_cycleswap.cpp

```diff
@@ -19,5 +19,25 @@
 using namespace std;
 
 int t08_cycleswap() {
+	int n;
+	cin >> n;
+	int mas[1000];
+	for (int i = 0; i < n; i++) {
+		cin >> mas[i];
+	}
+	int c = 0;
+	c = mas[0];
+	mas[0] = mas[n - 1];
+	int b = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'b' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4532058)